care/emr/api/viewsets/activity_definition.py

```diff
@@ -124,12 +124,24 @@ def validate_health_care_service(self, instance):
         ):
             raise ValidationError("Healthcare Service must be from the same facility")
 
+    def validate_data(self, instance, model_obj=None):
+        facility = self.get_facility_obj()
```

Comment:
If i call update API and replace the facility in the URL with another one, this will ignore the validation.